web-server/run.py

```diff
@@ -50,14 +46,15 @@ def sendwork():
     work = request.get_json()['work']
     name = _getRandomName()
 
-    db.containerstate.insert({"name":name, "state":"Pending"})
+    db.containerstate.insert_one({"name":name, "state":"Pending"})
 
     print("Creating job with work: ", work)
 
     try:
-        bus_service.send_queue_message(queueConf['queue_name'], Message({"name":name,"input":work}))
-    except Exception, e:
-        db.containerstate.update({"name":name},{"$set":{"state":"Error","message":traceback.format_exc()}})
+        sender = servicebus_client.get_queue_sender(queue_name=queueConf['queue_name'])
+        sender.send_messages(ServiceBusMessage(name=name,body=work))
```

Comment:
`ServiceBusMessage` doesn't take `name` as a keyword arguement. it will be ignored.

to send a `dict` data, please use `AmapAnnotatedMessage` instead of `ServiceBusMessage`, sample could be found here: https://github.com/Azure/azure-sdk-for-python/blob/main/sdk/servicebus/azure-servicebus/samples/sync_samples/send_and_receive_amqp_annotated_message.py#L52

Reply:
or you could convert the dict to a json string and pass it as the body of the `ServiceBusMessage`

Reply:
I have updated by your comments.